spec/system/jobseekers/jobseeker_applications_statuses_spec.rb

```diff
@@ -49,84 +49,5 @@
         expect(page).to have_css("#employment_history", text: I18n.t("shared.status_tags.incomplete"))
       end
     end
-
-    context "when the jobseeker has completed some details in their profile but not all of them" do
-      let!(:jobseeker_profile) { create(:jobseeker_profile, :with_qualifications, :with_employment_history, jobseeker: jobseeker, qualified_teacher_status: nil) }
-
-      it "shows all sections that have been filled in with the status tag 'in progress' and shows empty sections with the status tag 'not started'" do
-        visit job_path(vacancy)
-        within ".banner-buttons" do
-          click_on I18n.t("jobseekers.job_applications.banner_links.apply")
-        end
-
-        click_button "Start application"
-
-        expect(page).to have_css("#personal_details", text: I18n.t("shared.status_tags.incomplete"))
-        expect(page).to have_css("#professional_status", text: I18n.t("shared.status_tags.incomplete"))
-        expect(page).to have_css("#qualifications", text: I18n.t("shared.status_tags.incomplete"))
-        expect(page).to have_css("#training_and_cpds", text: I18n.t("shared.status_tags.incomplete"))
-        expect(page).to have_css("#employment_history", text: I18n.t("shared.status_tags.incomplete"))
-      end
-
-      context "when the jobseeker completes a section" do
```

Comment:
This whole flow is tested in spec/system/jobseekers/jobseekers_can_complete_a_job_application_spec.rb. I added assertions in there testing the status tags so that I could delete this test.